code/game/objects/items/manuals.dm

```diff
@@ -136,7 +136,7 @@
 				<h2>Basic ingredients preparation:</h2>
 
 				<b>Dough:</b> 10u water + 15u flour for simple dough.<br>
-				15u egg yolk + 15u flour + 5u sugar for cake batter.<br>
+				6u egg yolk + 8 egg white + 15u flour + 5u sugar for cake batter.<br>
```

Comment:
```suggestion
				6u egg yolk + 8u egg white + 15u flour + 5u sugar for cake batter.<br>
```
and shouldn't this be 6u egg yolk + 12u egg white if you want the creation of cake batter to still consume the entirety of two eggs?

Reply:
Cakes are 3 eggs. Yolks were reduced to 2u and whites 4. I just spilled my spaghetti and wrote 8 instead of 12 there for the whites.